.changeset/new-cougars-taste.md

```diff
@@ -0,0 +1,23 @@
+---
+'@clerk/backend': minor
+---
+
+Expose `totalCount` from `@clerk/backend` client responses for responses
+containing pagination information or for responses with type `{ data: object[] }`.
+
+
+Example:
+```typescript
+import { Clerk } from '@clerk/backend'
+
+const clerkClient = Clerk({ secretKey: '...'});
+
+// current
+const { data } = await clerkClient.organizations.getOrganizationList();
+console.log('totalCount: ', data.length);
+
+// new
+const { data, totalCount } = await clerkClient.organizations.getOrganizationList();
+console.log('totalCount: ', totalCount);
```

Comment:
Any idea how null is getting inferred ?

![image](https://github.com/clerk/javascript/assets/19269911/b1f0fb55-ed2e-4c0d-abd9-0f5462111ead)

Reply:
https://github.com/clerk/javascript/pull/2199/files#r1404230119

Reply:
If you are referring to our React hooks, then no. They only depend on FAPI.

`{ data: null, errors: Error[] }` makes sense.